.github/workflows/ci.yml

```diff
@@ -52,18 +52,17 @@ jobs:
         run: yarn build
   e2e:
     runs-on: ubuntu-latest
-    container: cypress/included:5.0.0
     strategy:
       matrix:
-        browser: [chrome] # cypress executable says 'Browser: 'firefox' was not found on your system or is not supported by Cypress.' seems to be defect
+        browser: [chrome, firefox]
+    container:
+      image: cypress/included:6.0.1
+      options: --user 1001
```

Comment:
Per official cypress git action example for Firefox: https://github.com/cypress-io/github-action#firefox